3.6.pylintrc

```diff
@@ -0,0 +1,24 @@
+
+[MESSAGES CONTROL]
+
+disable=misplaced-comparison-constant,no-self-use,consider-iterating-dictionary,
```

Comment:
can we list the suppressions in alphabetical order, one per line? it is hard to compare this list with the list for 2.7

Reply:
I agree. Do the suppressions map one to one to error codes? If so, can we add those as comments here so it's easier to refer to the error codes in the code?

Reply:
Sure. Working on this!

Reply:
I put both the symbol and the error code in the comments here.